package.json

```diff
@@ -126,8 +127,10 @@
     "i18next": "^21.8.8",
     "is-valid-coordinates": "^1.0.0",
     "jsonwebtoken": "^8.5.1",
+    "jszip": "^3.10.0",
```

Comment:
not another zip library please, we are including already node-stream-zip, wrapped by utils/file/fileZip.js